README.md

```diff
@@ -83,7 +83,50 @@ The CKAN core and base images can be built on Github actions by selecting the `B
 
 ## Deploying CKAN
 
-By default, merging a PR into main will trigger a build of the CKAN image and create PRs on the govuk-dgu-charts repo. Updating the tag will build and push a tagged image and create the PR for Staging and Production environments in the govuk-dgu-charts repo.
+### Environments and URLs
+
+- **Integration**: [https://www.integration.data.gov.uk/](https://www.integration.data.gov.uk/) (VPN required)
+- **Staging**: [https://www.staging.data.gov.uk/](https://www.staging.data.gov.uk/) (VPN required)
+- **Production**: [https://www.data.gov.uk/](https://www.data.gov.uk/) (public)
+
+---
+
+### Deploying to Integration
+
+1. **Make your change on a branch**
+   Develop your feature or fix on a dedicated branch.
+
+2. **Have the change approved**
+   Open a Pull Request, get it reviewed and approved.
+
+3. **Merge the change into `main`**
+   Once approved, merge your branch into `main`. This triggers a GitHub Action that:
+   - Builds a new CKAN Docker image.
+   - Creates a Pull Request in the [govuk-dgu-charts](https://github.com/alphagov/govuk-dgu-charts) repo, updating the CKAN image tag.
+
+4. **Merge the newly created PR**
+   In the govuk-dgu-charts repo, merge the auto-created PR. Your changes will then be deployed to **Integration** at:
+   [https://www.integration.data.gov.uk/](https://www.integration.data.gov.uk/)
```

Comment:
I may have misunderstood the question you asked. CKAN is the publishing app for DGU:
```suggestion
   [https://ckan.integration.publishing.service.gov.uk/](https://ckan.integration.publishing.service.gov.uk)
```